tests/models/jamba/test_modeling_jamba.py

```diff
@@ -632,11 +631,11 @@ def test_simple_batched_generate_with_padding(self):
         ).to(torch_device)
         out = self.model.generate(**inputs, do_sample=False, max_new_tokens=10)
         output_sentences = self.tokenizer.batch_decode(out)
-        self.assertEqual(output_sentences[0], EXPECTED_TEXTS[self.cuda_compute_capability_major_version][0])
-        self.assertEqual(output_sentences[1], EXPECTED_TEXTS[self.cuda_compute_capability_major_version][1])
+        self.assertEqual(output_sentences[0], output_sentences[0])
+        self.assertEqual(output_sentences[1], output_sentences[1])
```

Comment:
need update the `output_sentences` names on the right side.

Reply:
fixed, sorry for the naive mistake